private/bufpkg/bufimage/bufimageutil/bufimageutil_test.go

```diff
@@ -184,7 +180,7 @@ func TestTransitivePublic(t *testing.T) {
 	filteredImage, err := ImageFilteredByTypes(image, "c.Baz")
 	require.NoError(t, err)
 
-	_, err = desc.CreateFileDescriptorsFromSet(bufimage.ImageToFileDescriptorSet(filteredImage))
+	_, err = proto.Marshal(bufimage.ImageToFileDescriptorSet(filteredImage))
```

Comment:
Not sure if this is a suitable replacement for what was being exercised.

Reply:
Use the `protoencoding` package